zulipterminal/model.py

```diff
@@ -204,8 +204,9 @@ def _start_presence_updates(self) -> None:
             self.initial_data['presences'] = response['presences']
             self.users = self.get_all_users()
             if hasattr(self.controller, 'view'):
-                self.controller.view.users_view.update_user_list(
-                    user_list=self.users)
+                if self.controller.view.users_view.allow_update_user_list:
+                    self.controller.view.users_view.update_user_list(
+                        user_list=self.users)
```

Comment:
It seems a little strange having this logic in the model; wouldn't it be simpler in the view, where the variable is?

Reply:
Done :+1: 
I added a 
```if not self.allow_update_user_list and new_text == "":
            return
```
Inside `update_user_list`, which basically checks if user began search and if the function is not being called from `urwid.connect_signal` (no new alphabet entered to do search) - i.e the function must have been called from the thread to update user list (60s) and hence we can return safely.